web/mass_class/consumers.py

```
@@ -0,0 +1,517 @@
+            )
+        )
+
+    # Poll results event handler
+    async def poll_results(self, event):
+        await self.send(text_data=json.dumps({"type": "poll_results", "results": event["results"]}))
+
+    # Helper methods
+    def _is_teacher(self, user):
+        return user.courses_teaching.exists()
+
+    def _create_stream(self, user, session_id, sdp):
```

Ruff (0.8.2)

Missing return type annotation for private function `_create_stream`

Add return type annotation: `Optional[bool]`

(ANN202)